discovery/kubernetes/kubernetes.go

```diff
@@ -114,6 +115,7 @@ func (c *Role) UnmarshalYAML(unmarshal func(interface{}) error) error {
 type SDConfig struct {
 	APIServer          config.URL              `yaml:"api_server,omitempty"`
 	Role               Role                    `yaml:"role"`
+	KubeConfig         string                  `yaml:"kube_config"`
```

Comment:
We should verify when reloading the configuration that:
- The file exists (and validate it)
- We set the path relatively to the Prometheus config file (see the SetDiscovery method we use elsewhere)
- We do not specify multiple ways to connect to a kubernetes cluster.

Reply:
thank you give some suggestion, I will fix it about the path.

some a bit, I am not sure if this is necessary that check file existed, in fact clientcmd.BuildConfigFromFlags method will throw an error when load kubeconfig file.

Reply:
Okay, let's skip this for now. Can you fix the relative path? (see SetDirectory)